docs/advanced/settings.md

```diff
@@ -562,6 +562,13 @@ When the node becomes unavailable or is powered off, the VM only restarts and do
 
 **Default value**: **Harvester**
 
+:::caution
+
+This setting is part of the `settings.management.cattle.io` custom resource.
+Because of that you can't configure this setting, including `Logo`, `Favicon`, `Primary Color` and `Link Color`, using a [Harvester configuration](../install/harvester-configuration.md#system_settings) file.
+
+:::
```

Comment:
```suggestion
:::info important

Because this setting is part of the `settings.management.cattle.io` custom resource, you cannot configure any of the supported options (for example, `Logo` and `Primary Color` using a [Harvester configuration](../install/harvester-configuration.md#system_settings) file.

:::
```